src/index.html

```diff
@@ -7,8 +7,48 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+         src="images/logo.png"
+         alt="logo Moyo"
+         class="logo"
+        >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav item">
+            <a class="nav__link
+             is-active
+             apple">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link nav-1">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">SMARTPHONES</a>
+          </li>
```

Comment:
```suggestion
          </li>
          
          <li class="nav__item">
            <a class="nav__link nav-1">SAMSUNG</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link">SMARTPHONES</a>
          </li>
```
add empty lines between siblings